### PR TITLE
chore: anonymize send analytic properties

### PR DESCRIPTION
**Pull request by BZahory**
## **Description**

Certain send analytics can be used to correlated metric IDs with analytics data; this PR marks these as sensitive.

[Open in GitHub Codespaces](https://codespaces.new/MetaMask/metamask-extension/pull/26627?quickstart=1)

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Pre-merge author checklist**

- [ ] I've followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Extension Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by micaelae — APPROVED** [review]

File: ui/components/app/nfts-items/nfts-items.js [hunk 1]
```diff
@@ -169,12 +169,14 @@ export default function NftsItems({
         event: MetaMetricsEventName.sendAssetSelected,
         category: MetaMetricsEventCategory.Send,
         properties: {
-          ...sendAnalytics,
           is_destination_asset_picker_modal: false,
           new_asset_symbol: nft.name,
           new_asset_address: nft.address,
```
Comment (Gudahtt) [inline ui/components/app/nfts-items/nfts-items.js:174]: Should we move these as well? NFT address at least is an on-chain reference.

Comment (Gudahtt) [reply ui/components/app/nfts-items/nfts-items.js:174]: I might also be preferable to keep this data together in our metrics system but I'm not sure, depends on how its used.

File: ui/components/multichain/asset-picker-amount/asset-picker-modal/asset-picker-modal.tsx [hunk 2]
```diff
@@ -128,12 +128,14 @@ export function AssetPickerModal({
           event: MetaMetricsEventName.sendAssetSelected,
           category: MetaMetricsEventCategory.Send,
           properties: {
-            ...sendAnalytics,
             is_destination_asset_picker_modal: Boolean(isDest),
             new_asset_symbol: token.symbol,
             new_asset_address: token.address,
```
Comment (Gudahtt) [inline ui/components/multichain/asset-picker-amount/asset-picker-modal/asset-picker-modal.tsx:133]: Another reference here to move into `sensitiveProperties`

Comment (BZahory) [reply ui/components/multichain/asset-picker-amount/asset-picker-modal/asset-picker-modal.tsx]: Good call - fixed

File: ui/components/app/nfts-items/nfts-items.js (hunk 1 above)
Comment (BZahory) [reply ui/components/app/nfts-items/nfts-items.js]: We can move it, nfts can be pretty identifying at times if they're lower volume

**Review by Gudahtt — APPROVED** [review]
LGTM! Could you also submit this to `develop` at the earliest opportunity? So we don't accidentally undo this when we sync this release into develop later.

**Pull request merged**